Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -282,10 +297,19 @@ def domain_command(client, args):
             readable_output=md
         )
         command_results.append(result)
-
+        command_results.append(CommandResults(
+            outputs_prefix="DBotScore",
+            outputs_key_field=['Indicator', 'Vendor'],
+            outputs={
+                "Indicator": domain,
+                "Type": "domain",
+                "Vendor": "SecurityTrails",
+                "Score": 0
+            }
+        )
+        )
```

Comment:
I think instead of adding another CommandResults to the list a better coding would be adding the DBot object to the current CommandResults object this way: 

`dbot_score = Common.DBotScore(
            indicator=domain,
            indicator_type=DBotScoreType.DOMAIN,
            integration_name='SecurityTrails',
            score=Common.DBotScore.NONE
        )
   domain_indicator = Common.Domain(
        domain=domain,
        dbot_score=dbot_score
     )

    md = tableToMarkdown(f"Domain {domain}:", domain_data)
    result = CommandResults(
        outputs_prefix="Domain",
        outputs_key_field="Name",
        indicator=domain_indicator,
        outputs=domain_data,
        readable_output=md
    )
    command_results.append(result)`